src/templates/PoolV2/index.tsx

```diff
@@ -116,7 +117,12 @@ const Pool = () => {
   const updatedTabs = handleCheckTabs(tabs)
 
   function handleCheckTabs(tabsList: typeof tabs) {
-    if (parseFloat(pool?.fee_join_broker ?? '0') > 0 && tabs.length !== 6) {
+    const totalTabs = 7
+
+    if (
+      parseFloat(pool?.fee_join_broker ?? '0') > 0 &&
+      tabsList.length !== totalTabs
```

Comment:
Idk what tabList actually is, but probably should be checking if it's less than `totalTabs` or higher than `totalTabs`, you should check if you really just want `!==`